app/assets/javascripts/.eslintrc.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   extends: ['airbnb-base/legacy', 'prettier'],
   parserOptions: {
-    ecmaVersion: 2017,
+    ecmaVersion: 2018,
```

Comment:
Hi @ludamillion, did you check *all* files in `app/assets/javascripts` when you said "does not introduce new linting errors" or just `sendFetch`?

Reply:
I checked all the files in `app/assets/javascripts`. To be specific I ran the linter on that directory before and after the change and diffed the outputs. Yeah, I was sure to double check that I didn't introduce anything new as that's always the worry when making a change like this.

Aside from the change I mentioned regard the short circuit `&&` the only difference in the diffs was the removal of the error in `sendFetch`

Reply:
Great for the clarification! Thank you!